articles/appliance/index.md

```diff
@@ -2,57 +2,57 @@
+###On-premises deployment communication with Auth0 
 
-No information is transferred outside the Auth0 nodes without consent from you. Nodes will contact external endpoints for maintenance only with your consent and supervision (e.g. to download a new version of the software, or apply an security patch).
+No information will be transferred outside the Auth0 nodes without consent from you. For maintenance purposes, nodes will contact external endpoints only with your consent and supervision (e.g. in order to download a new version of the software or apply an security patch).
 
-###Does Auth0 staff have access to the nodes?
+###Access to nodes by Auth0
 
-Not without your consent.
+The Auth0 staff does not have access to any nodes without your consent.
 
-###How often is maintenance performed?
+###Scheduled maintenance
 
-The typical update cycle is once per month. You can control the schedule of updates applied.
+The typical update cycle is once per month. You can also customize the update schedule.
 
-###What does maintenance consist of?
+###Maintenance access
 
-We will remotely access your nodes (e.g. via temporary SSH or through remote control software) and use the configuration dashboard to manage the nodes. Occasionally we might perform lower level actions on the nodes. All these happen under your observation. We will notify in advance the nature of the activities we will perform, and provide instructions on actions you might need to perform on your side.
+To perform maintenance, Auth0 will remotely access your nodes (either through a temporary SSH or through remote control software) and refer to the configuration dashboard to manage the nodes. Occasionally, lower-level actions may be performed on the nodes. All events will occur under your observation. You will be notified in advance on the nature of activities to be performed, and provided instructions on any actions you may need to execute on your side.
 
-###How are nodes monitored?
+###Node monitoring
 
-Because nodes are running behind your firewall, under your control, and frequently beyond our reach, you are responsible for monitoring that the service is performing to expected levels. We recommend VM level monitoring (e.g. CPU utilization) as well as service level monitoring. Auth0 provides specific monitoring endpoints you can attach to your own monitoring tools (e.g. Microsoft System Center, IBM Tivoli, HP OpenView, etc.). Detailed guidance is available [here](/monitoring).
+Because nodes are running under your control behind your firewall and therefore beyond our reach, you are responsible for monitoring service performance. We recommend VM level monitoring (e.g. CPU utilization) as well as service level monitoring. Auth0 provides specific monitoring endpoints that you can attach to your own monitoring tools (e.g. Microsoft System Center, IBM Tivoli, HP OpenView, etc.). Detailed guidance is available at [Monitoring Auth0](/monitoring).
```

Comment:
performing to expected level is not the same thing as performance - this also relates to uptime for example